plugins/woocommerce/includes/admin/settings/class-wc-settings-advanced.php

```diff
@@ -392,11 +392,19 @@ protected function get_settings_for_legacy_api_section() {
 					'id'    => 'legacy_api_options',
 				),
 				array(
-					'title'   => __( 'Legacy API', 'woocommerce' ),
-					'desc'    => __( 'Enable the legacy REST API', 'woocommerce' ),
-					'id'      => 'woocommerce_api_enabled',
-					'type'    => 'checkbox',
-					'default' => 'no',
+					'title'    => __( 'Legacy API', 'woocommerce' ),
+					'desc'     => __( 'Enable the legacy REST API', 'woocommerce' ),
+					'id'       => 'woocommerce_api_enabled',
+					'type'     => 'checkbox',
+					'default'  => 'no',
+					'desc_tip' => sprintf(
+						// translators: Placeholder is a URL.
+						__(
+							'⚠<b>️ The Legacy REST API will be removed in WooCommerce 9.0.</b> A separate WooCommerce extension will be available to re-enable it. <b><a target=”_blank” href="%s">Learn more about this change</a></b>',
```

Comment:
```suggestion
							'⚠️ <b>️The Legacy REST API will be removed in WooCommerce 9.0.</b> A separate WooCommerce extension will soon be available to keep it enabled. <b><a target=”_blank” href="%s">Learn more about this change</a></b>',
```

Thinking that this alternate wording (applies to the other spot as well) might land a little better for people using the Legacy API because, A) "soon" makes it more clear that the plugin will be available _before_ the API is removed, and B) helps them understand that there won't be any interruption in their service if they get that plugin.

(Also thought that switching the warning symbol to the emoji version might stand out a bit more... but that's just a personal preference.)

Reply:
Makes sense, thanks. Also I didn't even know there were "emoji and non-emoji" variants of the warning sign.